node/pkg/ethereum/watcher.go

```diff
@@ -99,6 +99,9 @@ type (
 		// Minimum number of confirmations to accept, regardless of what the contract specifies.
 		minConfirmations uint64
 
+		// Number of blocks we will wait until we abandon a previously observed message that is not on the current chain anymore
+		messageBlocksTillAbandon uint64
```

Comment:
There is quite a variety of values being passed in for messageBlocksTillAbandon (as opposed to before when it was four times minConfirmations). Is there logic to how you came up with these values? If so, could you put a comment explaining how to come up with the value?